operator/pkg/translate/translate_value_test.go

```diff
@@ -283,37 +276,11 @@ values:
 	}
 }
 
-func TestNewReverseTranslator(t *testing.T) {
-	tests := []struct {
-		name         string
-		minorVersion version.MinorVersion
-		wantVer      string
-		wantErr      bool
-	}{
-		{
-			name:         "version 1.4",
-			minorVersion: version.NewMinorVersion(1, 4),
-			wantVer:      "1.4",
-			wantErr:      false,
-		},
-		// TODO: implement 1.5 and fallback logic.
-		{
-			name:         "version 1.99",
-			minorVersion: version.NewMinorVersion(1, 99),
-			wantVer:      "",
-			wantErr:      true,
-		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			got, err := NewReverseTranslator(tt.minorVersion)
-			if (err != nil) != tt.wantErr {
-				t.Errorf("NewReverseTranslator() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-			if got != nil && tt.wantVer != got.Version.String() {
-				t.Errorf("NewReverseTranslator() got = %v, want %v", got.Version.String(), tt.wantVer)
-			}
-		})
+// errToString returns the string representation of err and the empty string if
+// err is nil.
+func errToString(err error) string {
```

Comment:
a bit hard to tell from the diff .... is this used?

Reply:
Yes, it was in another file that was removed before, just copying it over here for some tests here that use it